src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  &__style {
```

Comment:
```suggestion 
```
Why do you need this `header__style` classes you can apply styles directly to `.header`

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/200179531-9138e3d1-d998-4d9c-96b4-8d2a04d75bfe.png)